vm-docker/Dockerfile

```diff
@@ -0,0 +1,117 @@
+    g++ \
+    less \
+    libbz2-dev \
+    libcurl4-openssl-dev \
+    liblzma-dev \
+    libncurses5-dev \
+    pkg-config \
+    python3-venv \
+    unzip \
+    wget \
+    zlib1g-dev \
+    && \
+	# clean up apt cache
+    rm -rf /var/lib/apt/lists/*
+
+RUN mkdir home/tools 
+
+# Install htslib
+ENV SAMTOOLS_VERSION="1.11"
+RUN cd home/tools \
+    && wget https://github.com/samtools/htslib/releases/download/${SAMTOOLS_VERSION}/htslib-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& tar xjf htslib-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& rm htslib-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& cd htslib-${SAMTOOLS_VERSION} \
+	&& ./configure \
+	&& make \
+	&& make install \
+	&& make clean
+
+
+# Install bcftools
+RUN cd home/tools \
+    && wget https://github.com/samtools/bcftools/releases/download/${SAMTOOLS_VERSION}/bcftools-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& tar xjf bcftools-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& rm bcftools-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& cd bcftools-${SAMTOOLS_VERSION} \
+	&& ./configure \
+	&& make \
+	&& make install \
+	&& make clean
+
+# Install samtools
+RUN cd home/tools \
+    && wget https://github.com/samtools/samtools/releases/download/${SAMTOOLS_VERSION}/samtools-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& tar xjf samtools-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& rm samtools-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& cd samtools-${SAMTOOLS_VERSION} \
+	&& ./configure \
+	&& make \
+	&& make install \
+	&& make clean
+
+
+# install vcftools
+ENV VCFTOOLS_VERSION="0.1.16"
+RUN cd home/tools \
+    && wget https://github.com/vcftools/vcftools/releases/download/v${VCFTOOLS_VERSION}/vcftools-${VCFTOOLS_VERSION}.tar.gz \
+    && tar xf vcftools-${VCFTOOLS_VERSION}.tar.gz \
+    && rm vcftools-${VCFTOOLS_VERSION}.tar.gz \
+	&& cd vcftools-${VCFTOOLS_VERSION} \
+	&& ./configure \
+	&& make \
+	&& make install \
+	&& make clean
+
+# install picard
+ENV PICARD_VERSION="2.25.0"
+RUN cd home/tools \
+    && wget https://github.com/broadinstitute/picard/releases/download/${PICARD_VERSION}/picard.jar 
+
+# install GATK
+ ENV GATK_VERSION="4.2.0.0"
+ RUN cd home/tools \
+    && wget https://github.com/broadinstitute/gatk/releases/download/${GATK_VERSION}/gatk-${GATK_VERSION}.zip \
+    && unzip gatk-${GATK_VERSION}.zip \
+    && rm gatk-${GATK_VERSION}.zip 
+
+
+# Install python packages
+RUN apt-get update \
+	&& apt-get dist-upgrade -y \
+	&& apt-get install -y \
+    g++ \
+    libc6-dev \
+    libffi-dev \
+    libgdbm-dev \
+    liblapack-dev \
+    liblapack3 \ 
+    libncursesw5-dev \
+    libopenblas-base \
+    libopenblas-dev \
+    libsqlite3-dev \
+    libssl-dev \
+    openssl \
+    python-smbus \
+    python3 python-pip 
+
+# Upgrade pip to latest version
+RUN python3 -m pip install --upgrade pip
+
+# Install hail
+ENV HAIL_VERSION="0.2.64"
+RUN python3 --version
+RUN python3 -m pip install wheel pypandoc hail==${HAIL_VERSION}
+
+# Install R
+ENV R_VERSION="3.5.2-1"
+RUN apt-get install -y \
+    r-base=${R_VERSION} \
+    r-base-dev=${R_VERSION} 
+
```

Comment:
Depending on how this image will be used, may want to also pre-install `bedtools` (https://github.com/broadinstitute/str-callers/blob/master/docker/base-other-tools/Dockerfile#L17-L26) 
and python libs like 
`scipy numpy pandas matplotlib seaborn ipython pybedtools`

Reply:
I'll add bedtools and those python libraries - they all make sense to me and I believe are on many peoples init scripts.